iothub/device/tests/DeviceClientTests.cs

```diff
@@ -157,6 +157,32 @@ public async Task DeviceClient_OperationTimeoutInMilliseconds_Equals_0_Receive()
             await innerHandler.Received().ReceiveAsync(Arg.Any<TimeoutHelper>()).ConfigureAwait(false);
         }
 
+        [TestMethod]
+        public async Task DeviceClient_OnMethodCalled_Unsubscribe()
+        {
+            // arrange
+            DeviceClient deviceClient = DeviceClient.CreateFromConnectionString(fakeConnectionString);
+            var innerHandler = Substitute.For<IDelegatingHandler>();
+            deviceClient.InnerHandler = innerHandler;
+
+            // act
+            await deviceClient
```

Comment:
this call can be skipped for this test, right?

Reply:
It is required since we need to enabled before disabling. I'll check on this though.